### PR TITLE
Basic storage interaction

### PR DESCRIPTION
**Pull request by Zindre17**

### CODE REVIEW

File: Startup.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.OpenApi.Models;
+using This_is_variant_music_api.Storage;
 
 namespace This_is_variant_music_api
```
Comment (adriansberg) [inline Startup.cs]: What did we decide on regarding casing convention for namespace?

**Review by adriansberg — APPROVED** [review]
LGTM 👌

File: Startup.cs (hunk 1 above)
Comment (Zindre17) [reply Startup.cs]: I think PascalCase is the norm for namespaces. Maybe something simpler, like `VariantMusicApi`?

Comment (adriansberg) [reply Startup.cs]: Yes, that's better 👌

**Comment by Zindre17** [comment]
Oops. Glemte å bytte target branch til main etter jeg merga target branchen 😅 Oppretter ny PR med riktig target...

**Pull request merged**